src/utils/sendUpdates.ts

```diff
@@ -80,9 +103,7 @@ export default class Updates {
         messageUpdates.channelId.send({
           embeds: [await this.messageWikiStyle(e, messageUpdates.url)],
         })
-        if (messageUpdates.channelType === ChannelTypes.PROD) {
-          await this.twitter.tweetWikiActivity(e, messageUpdates.url)
-        }
+        await this.checkAndTweet(messageUpdates, e)
```

Comment:
lets avoid var names like "e" and use something that can explain or give some hints about what to expect from the var

 we dont need to pay extra for a variable name like "event" or "activity" :)))

Reply:
True ! i will add a commit to fix it.